### ISSUE TITLE
Namespaces

### ISSUE THREAD

**Issue by eduherminio** [afimb/siri-validator#2]
First of all, thanks for the tool.
I'm facing just a problem using it:

The validator generates the following namespace:
`xmlns:ns6="http://wsdl.siri.org.uk">`
but in order to make it work with my server, I need to change it to
`xmlns:ns6="http://wsdl.siri.org.uk/siri">`

I've tried substituting all fields containing that string in `SiriConsumer.java` file before compiling it so that it generates proper XMLs for my purpose, but the namespaces generated after compiling the tool doesn't seem to change.

Any feedback is welcome.
Thanks in advance,

Eduardo

**Comment by metienne** [comment]
Hello,
can you give me more informations and examples to show your problem?
On which SIRI profile are you working?
The classes generating the xml are produced by https://github.com/afimb/siri_api project and used by this one.

**Comment by eduherminio** [comment]
I've just found [here](https://github.com/afimb/siri_api/blob/master/siri_wsdl/src/wsdl/ReadMe.txt) the source of my problem:
```
2013.06.26 French comments (compatibility issue)
 
* [fr]  Change SOAP Web Service namesapce to http://wsdl.siri.org.uk (previously http://wsdl.siri.org.uk/siri) 
```
I am using SIRI v2.0 XSD, dowloaded from [SIRI official site](http://user47094.vs.easily.co.uk/siri/schema/schemas.htm), where `http://wsdl.siri.org.uk/siri` is still used as namespace.
That explains those minor modifications I have to make in either your validator or my XSD/code so that I can use it properly.

**Comment by metienne** [comment]
Can you check the version of your siri XSD, it is in the ReadMe.txt file
our version is 2.0o
2015.05.13 Bug fix (from SG7 feedback) SIRI 2.0o
I have a look on the Siri Official site and I just found an old version from 2013.02.20; this site is not up to date (last change on 2013/05/21)
the namespace has changed on 2013.06.26 for backward compatibility with SIRI 1.x

you can download latest XSD on https://www.vdv.de/siri.aspx site or on our github project :  https://github.com/afimb/siri_api

**Comment by eduherminio** [comment]
Oh, the official site is not up to date, it's just... fantastic.
I have been working with 2013.02.20 old version, indeed, so I'd better get ready for a SIRI version migration.
Thanks a lot for your help, Michel.
